AUTHORS

```diff
@@ -3567,5 +3567,7 @@ Tim Shilov <tim@shilov.dev>
 Obiwac <obiwac@gmail.com>
 Yu Gu <guyu2876@gmail.com>
 andreysoktoev <andrey.soktoev@gmail.com>
+Pavel Horal <pavel.horal@orchitech.cz>
+Claudio Wunder <cwunder@hubspot.com>
```

Comment:
Duplicate, needs a .mailmap entry.

Reply:
@ovflowd Which email should we use in the AUTHORS file? hubspot.com or gnome.org?

Reply:
(I went with gnome.org but it's easy enough to change at any time.)